src/ordering/ordering_driver.cpp

```diff
@@ -37,24 +37,23 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #include "cpp_common/pgdata_getters.hpp"
 #include "cpp_common/alloc.hpp"
 #include "cpp_common/assert.hpp"
-#include "c_types/ii_t_rt.h"
 
 #if 0
 #include "ordering/ordering.hpp"
 #endif
 
 
-void 
-pgr_do_ordering(
+void
+do_ordering(
```

Comment:
you get this kind of [error](https://github.com/pgRouting/GSoC-pgRouting/actions/runs/16076368891/job/45372443440#step:8:16): 
```
 ERROR:  could not load library "/usr/lib/postgresql/16/lib/libpgrouting-4.0.so": 
 /usr/lib/postgresql/16/lib/libpgrouting-4.0.so: undefined symbol: _Z11do_orderingPKciPlPmPPcS4_S4_
```

Because the implementation does not match the definition.

https://github.com/pgRouting/GSoC-pgRouting/pull/460/files#diff-c78780c857dc6a395eed58eab37e1edbee777cbc83eca43be4e3e57971ff36ae